src/extension/options-page/DashboardDialogs/Wallet.tsx

```diff
@@ -629,7 +629,20 @@ export function DashboardWalletHistoryDialog(
 
     const [redPacketRecords, setRedPacketRecords] = useState<RedPacketRecord[]>([])
     const inboundRecords = redPacketRecords.filter((record) => record.claim_address === wallet.address)
-    const outboundRecords = redPacketRecords.filter((record) => record.sender_address === wallet.address)
+    // const outboundRecords = redPacketRecords.filter((record) => record.sender_address === wallet.address)
+
+    // region HACK: THIS TEMPORARY CODE
```

Comment:
Why do you say so?

Reply:
i need redesign data structure,
match maskbook current wallet type system.